### PR TITLE
WireServerEndpoint changes

### PR DESCRIPTION
**Pull request by Jyotsna-Anand**
Initial PR review from private branch - https://github.com/Azure/azure-linux-extensions/pull/1207

### CODE REVIEW

File: VMEncryption/main/EncryptionSettingsUtil.py [hunk 1]
```diff
@@ -227,14 +227,34 @@ def get_http_util(self):
...
+        wireserver_endpoint_file = CommonVariables.wireserver_endpoint_file
+        wireserver_IP = None
+        wireserver_endpoint_uri = CommonVariables.wireserver_endpoint_uri
+
+        if os.path.exists(wireserver_endpoint_file):
+            with open(wireserver_endpoint_file, 'r') as wip:
+                wireserver_IP = wip.read()
+                self.logger.log("wireserver_IP found in {0} = {1}".format(wireserver_endpoint_file, wireserver_IP))
+
+        if wireserver_IP == None:
```
Comment (vermashi) [inline VMEncryption/main/EncryptionSettingsUtil.py:241]: style: use "is None" instead of "== None"

**Review by vermashi — APPROVED** [review]

File: VMEncryption/main/Common.py [hunk 2]
```diff
@@ -47,7 +47,9 @@ class CommonVariables:
     encryption_settings_counter_file = 'counter.txt'
     encryption_settings_counter_path = encryption_key_mount_point + '/' + encryption_settings_counter_file
 
-    wireserver_endpoint = "http://169.254.169.254:80/machine?comp=diskEncryptionData"
+    static_wireserver_IP = '168.63.129.16'
```
Comment (ejarvi) [inline VMEncryption/main/Common.py:50]: consider adding a code comment pointing to this documentation on 168.63.129.16 
https://docs.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16

File: VMEncryption/main/EncryptionSettingsUtil.py (hunk 1 above)
Comment (ejarvi) [reply VMEncryption/main/EncryptionSettingsUtil.py:241]: FWIW https://www.python.org/dev/peps/pep-0008/

**Review by ejarvi — CHANGES_REQUESTED** [review]

**Review by ejarvi — APPROVED** [review]

File: VMEncryption/main/EncryptionSettingsUtil.py [hunk 3]
```diff
@@ -227,14 +227,39 @@ def get_http_util(self):
...
 
+    def get_wireserver_endpoint_uri(self):
+
+        wireserver_endpoint_file = CommonVariables.wireserver_endpoint_file
+        wireserver_IP = None
+        wireserver_endpoint_uri = CommonVariables.wireserver_endpoint_uri
+
+        if os.path.exists(wireserver_endpoint_file):
+            with open(wireserver_endpoint_file, 'r') as wip:
+                wireserver_IP = wip.readline().rstrip()
```
Comment (vimish) [inline VMEncryption/main/EncryptionSettingsUtil.py:238]: Use strip() instead of rstrip() ?

**Review by vimish — APPROVED** [review]
LGTM. 
Just a minor suggestion added.

**Pull request merged**